src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -52,8 +57,49 @@ public static MvcForm BeginForm(this IHtmlHelper htmlHelper, FormMethod method)
                 throw new ArgumentNullException(nameof(htmlHelper));
             }
 
-            return htmlHelper.BeginForm(actionName: null, controllerName: null, routeValues: null,
-                                        method: method, htmlAttributes: null);
+            return htmlHelper.BeginForm(
+                actionName: null,
+                controllerName: null,
+                routeValues: null,
+                method: method,
+                suppressAntiforgery: false,
+                htmlAttributes: null);
+        }
+
+        /// <summary>
+        /// Renders a &lt;form&gt; start tag to the response. When the user submits the form, the
+        /// current action will process the request.
+        /// </summary>
+        /// <param name="htmlHelper">The <see cref="IHtmlHelper"/> instance this method extends.</param>
+        /// <param name="method">The HTTP method for processing the form, either GET or POST.</param>
+        /// <param name="htmlAttributes">
+        /// An <see cref="object"/> that contains the HTML attributes for the element. Alternatively, an
+        /// <see cref="System.Collections.Generic.IDictionary{string, object}"/> instance containing the HTML
+        /// attributes.
+        /// </param>
+        /// <returns>
+        /// An <see cref="MvcForm"/> instance which renders the &lt;/form&gt; end tag when disposed.
+        /// </returns>
+        /// <remarks>
+        /// In this context, "renders" means the method writes its output using <see cref="ViewContext.Writer"/>.
+        /// </remarks>
```

Comment:
This method is showing up as "added" in the diff, but it's unchanged except for formatting.